pkg/volume/local/local.go

```diff
@@ -307,3 +359,43 @@ func (u *localVolumeUnmounter) TearDownAt(dir string) error {
 	glog.V(4).Infof("Unmounting volume %q at path %q\n", u.volName, dir)
 	return util.UnmountMountPoint(dir, u.mounter, true) /* extensiveMountPointCheck = true */
 }
+
+// Block Volume
```

Comment:
nit: This comment isn't adding much value. I recommend removing or expanding it.

Reply:
Fixed.